$MirOS$
$MirOS: ports/devel/pango/patches/patch-configure_in,v 1.1.7.1 2005/03/18 15:45:22 tg Exp $
--- configure.in.orig	Tue Aug 16 01:04:53 2005
+++ configure.in	Thu Sep 22 21:42:48 2005
@@ -199,7 +199,7 @@ if test x$no_x = xyes ; then
   AC_MSG_WARN([X development libraries not found])
   have_x=false
 else
-  X_LIBS="$X_LIBS -lX11"
+  X_LIBS="$X_LIBS -lX11 -lXext"
   have_x=true
   AC_DEFINE(HAVE_X, 1, [Have X libraries])
 fi
@@ -231,7 +231,7 @@ if $have_fontconfig ; then
     LDFLAGS=$pango_save_ldflags
   fi
 
-  FREETYPE_LIBS="$FONTCONFIG_LIBS $FREETYPE_LIBS"
+  FREETYPE_LIBS="$FONTCONFIG_LIBS -lexpat $FREETYPE_LIBS"
   FREETYPE_CFLAGS="$FONTCONFIG_CFLAGS $FREETYPE_CFLAGS"
 
   AC_SUBST(FREETYPE_LIBS)
@@ -393,6 +393,8 @@ for module in $included_modules; do
   else
     dir=`echo $module | sed "s/-.*//"`
   fi
+
+  XFT_LIBS="$XFT_LIBS -lXext"
 
   if echo $module | egrep -- "-fc($|,)" > /dev/null; then
     INCLUDED_FC_MODULES="$INCLUDED_FC_MODULES \$(top_builddir)/modules/$dir/libpango-$module.la"
